.eslintrc.js

```diff
@@ -44,6 +44,13 @@ module.exports = {
             "error",
             "never"
         ],
-        "no-var": "error"
+        "no-var": "error",
+        // part of Node Style-guide but ignored
+        "max-len": 0,
+        "max-statements": 0,
+        "curly": ["error", "multi-or-nest"],
```

Comment:
@tchakabam I think it's better to use the [consistent](https://eslint.org/docs/rules/curly#consistent) style, instead of `multi-or-nest`. This way we prevent using brace-less statements chained to brace blocks, such as `if/else`.

Reply:
afaiu `consistent` is in addition to any of the `multi..` options (`"When using any of the multi* options, you can add an option to enforce all bodies of a if, else if and else chain to be with or without braces."`)

let's try that! :)